From 391d2765904fb761deefc4d07c1b1c41cd9b8314 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 17 Oct 2016 11:05:33 -0400
Subject: [PATCH] RELENG-612 prefer (git+)https urls Reviewed by: Trent Mick
 <trent.mick@joyent.com> Approved by: Trent Mick <trent.mick@joyent.com>

---
 .gitmodules  | 6 +++---
 package.json | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index dbadc87..1d72cc4 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,9 +1,9 @@
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
diff --git a/package.json b/package.json
index 8a31a32..84008de 100644
--- a/package.json
+++ b/package.json
@@ -8,10 +8,10 @@
         "async": "0.1.18",
         "backoff": "2.3.0",
         "bunyan": "1.3.2",
-        "sdc-bunyan-serializers": "git+ssh://git@github.com:joyent/sdc-bunyan-serializers.git#aefc119",
+        "sdc-bunyan-serializers": "git+https://github.com/joyent/sdc-bunyan-serializers.git#aefc119",
         "nodeunit": "0.7.4",
         "restify": "2.7.0",
-        "sdc-clients": "git+ssh://git@github.com:joyent/node-sdc-clients.git#c962959"
+        "sdc-clients": "git+https://github.com/joyent/node-sdc-clients.git#c962959"
     },
     "sdcDependencies": {
         "config-agent": ">=1.3.0"
-- 
2.21.0

